Assignments/stu7/assignments.py

```diff
@@ -103,4 +102,14 @@ def ex7():
 #
 # Place your functions here...
 #
+# Ex1:
+def sort_people(people_list, a, b):
+    if b == "asc":
```

Comment:
We will probably need another if to handle the "desc" direction, right?

Reply:
it has been added